var/spack/repos/builtin/packages/py-holland/package.py

```diff
@@ -0,0 +1,23 @@
+# Copyright 2013-2020 Lawrence Livermore National Security, LLC and other
+# Spack Project Developers. See the top-level COPYRIGHT file for details.
+#
+# SPDX-License-Identifier: (Apache-2.0 OR MIT)
+from spack import *
+
+
+class PyHolland(PythonPackage):
+    """Holland is an Open Source backup framework originally developed
+    by Rackspace and written in Python. Its goal is to help facilitate
+    backing up databases with greater configurability, consistency, and ease.
+    Holland currently focuses on MySQL, however future development will include
+    other database platforms and even non-database related applications.
+    Because of its plugin structure, Holland can be used to backup anything
+    you want by whatever means you want."""
+
+    homepage = "http://hollandbackup.org/"
+    url      = "https://github.com/holland-backup/holland/archive/1.2.2.tar.gz"
+
+    version('1.2.2', sha256='836337c243b2dff5ff6a3ce0b647f123ab24697a5de8ac8ae8b7839aa23dff68')
+
+    depends_on('py-setuptools', type='build')
```

Comment:
```suggestion
    depends_on('py-setuptools', type=('build', 'run'))
```
It's listed in `install_requires`, so there's a good chance that the package installs some scripts that import `pkg_resources`

Reply:
Fixed in https://github.com/spack/spack/pull/19924/commits/46f46ab39d18cc0ec7c4e56fdbffaa4e5f008e63